File: 10951 A+B - 4/10951.js
let fs = require('fs');
let input = fs.readFileSync('/dev/stdin').toString().split('\n');

let inputCount = input.length;
for (let i = 0; i < inputCount - 1; ++i) {
    let inputs = input[i].split(' ');
    let a = parseInt(inputs[0]);
    let b = parseInt(inputs[1]);

    console.log(a + b);
}

File: 10951 A+B - 4/10951.java
import java.io.*;

public class Main {
    static public void main(String[] args) throws IOException {
        BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bufferedWriter = new BufferedWriter(new OutputStreamWriter(System.out));

        String inputLine;
        while ((inputLine = bufferedReader.readLine()) != null) {
            String[] inputs = inputLine.split(" ");
            int a = Integer.parseInt(inputs[0]);
            int b = Integer.parseInt(inputs[1]);

            bufferedWriter.write(String.format("%d\n", a + b));
        }

        bufferedWriter.flush();
    }
}
